sdk/test/Services/DynamoDBv2/IntegrationTests/JSONTests.cs

```diff
@@ -95,7 +95,7 @@ public void TestDocumentPutGet()
 
         [TestMethod]
         [TestCategory("DynamoDBv2")]
-        public static void TestDocumentJsonConversions()
```

Comment:
@normj These tests weren't running. It looks like the methods cannot be static, this was the message on our build system:
> [MSTest][Discovery][AWSSDK.IntegrationTests.NetFramework.dll] UTA007: Method TestDocumentBinaryDecoding defined in class AWSSDK_DotNet.IntegrationTests.Tests.DynamoDB.DynamoDBTests does not have correct signature. Test method marked with the [TestMethod] attribute must be non-static, public, return-type as void  and should not take any parameter. Example: public void Test.Class1.Test(). Additionally, if you are using async-await in test method then return-type must be Task. Example: public async Task Test.Class1.Test2()

I also had to fix the sample JSON used as it was complaining about trailing commas.